features/formatter/use_the_tap_formatter.feature

```diff
@@ -0,0 +1,92 @@
+Feature: Use the TAP formatter
+  So that I can get non-XML parseable results
+  As a Developer
+  I need to be able to use a TAP formatter
+
+  Scenario: Empty spec causes various result states
+    Given the spec file "spec/Formatter/TapExample1/TapSpec.php" contains:
+      """
+      <?php
+
+      namespace spec\Formatter\TapExample1;
+
+      use PhpSpec\ObjectBehavior;
+
+      class TapSpec extends ObjectBehavior
+      {
+          function it_is_most_definitely_pending()
+          {
+          }
+
+          function it_is_most_definitely_passing()
+          {
+            $this->fire('pass')->shouldReturn('pass');
+          }
+
+          function it_is_most_definitely_failing()
+          {
+            $this->fire('fail')->shouldReturn('pass');
+          }
+
+          function it_is_most_definitely_broken()
+          {
+            $this->fire('broken')->shouldReturn('pass');
+          }
+
+          function it_is_most_definitely_skipping()
+          {
+            $this->fire('skip')->shouldReturn('pass');
+          }
+      }
+      """
+    And the class file "src/Formatter/TapExample1/Tap.php" contains:
+      """
+      <?php
+
+      namespace Formatter\TapExample1;
+
+      use PhpSpec\Exception\Example\SkippingException;
+      use PhpSpec\Exception\Example\ErrorException;
+
+      class Tap
+      {
+          public function fire($stuff)
+          {
+              switch ($stuff) {
+                case 'pass':
+                return 'pass';
+                break;
+                case 'fail':
+                return 'fail';
+                break;
+                case 'broken':
+                throw new ErrorException('error','something terrible occurred','foo.php',99);
+                case 'skip':
+                throw new SkippingException('php is not installed');
```

Comment:
the SkippingException is meant to be used in spec examples, not in the code. Please make the scenario reflect the real usage of the exception